lib/src/annotation/annotation_manager.dart

```diff
@@ -75,14 +75,12 @@ class AnnotationManager {
 
 /// The super class for all AnnotationManagers.
 class BaseAnnotationManager {
-  BaseAnnotationManager(
+  BaseAnnotationManager._(
       {required String id,
       required BinaryMessenger messenger,
-      String? channelSuffix})
+      required String channelSuffix})
```

Comment:
nit: `channelSuffix` is not stored here anymore, might as well be removed from here. Subclasses might as well declare their own constructor parameter: `required super.channelSuffix` > `required String channelSuffix`)